ci/docker/kpsrtutorial-1.0.0.dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ghcr.io/klepsydra-technologies/kpsrbase:1.0.0
+
+# Install kpsr-core
+ARG KPSRCOREVERSION="v8.0.0"
+RUN git clone https://github.com/klepsydra-technologies/kpsr-core.git && \
+    cd kpsr-core && \
+    git checkout ${KPSRCOREVERSION} && \
+    git submodule update --init --recursive && \
+    mkdir build && cd build && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_YAML=true -DKPSR_WITH_CODE_METRICS=true ../ && \
+    make -j$(nproc) && \
+    sudo make install && \
+    cd ../../ && \
+    rm -rf kpsr-core
+
+# Install Python package build needed by kpsr-codegen
+RUN pip3 install build
+# Install kpsr-codegen
+ARG KPSRCODEGENVERSION="main"
+RUN git clone https://github.com/klepsydra-technologies/kpsr-codegen.git && \
+    cd kpsr-codegen && \
+    git checkout ${KPSRCODEGENVERSION} && \
+    sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules && \
```

Comment:
As the repo kpsr-codegen uses ssh for its submodules (check [here](https://github.com/klepsydra-technologies/kpsr-codegen/blob/main/.gitmodules)) I choose to use `sed` to replace the ssh by https clone.

We could either keep the `sed` or update the kpsr-codegen repo.

Reply:
Done in 9627054852060974f110663e10d69acb0952a73f